submissions/7SergSmith7/Memory-pair-game/app.js

```diff
@@ -0,0 +1,105 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
+
+const images = [
+  "acrobatics",
+  "boneCutter",
+  "chaosRing",
+  "doubleStrike",
+  "innerDemon",
+  "metamorphosis",
+  "splittingSoul",
+  "manaBurn",
+];
+const pairCounter = 8;
+const winMsg = "You Win Bro! Play again?";
+const cards = [...images, ...images];
+const gameField = document.querySelector(".game-field");
+
+let firstCard = "";
+let secondCard = "";
+let pairsCounter = 0;
+
+function cardTemplate(image) {
```

Comment:
Start your function names from a verb, please
```suggestion
function generateCardTemplate(image) {
```